src/napistu/network/napistu_graph_core.py

```diff
@@ -114,7 +179,7 @@ def reverse_edges(self) -> "NapistuGraph":
             f"Reversed graph edges. Current state: reversed={self._metadata['is_reversed']}"
         )
 
-        return self
+        return None
```

Comment:
Ah the much debated return in case of in-place modification.

While for independent functions I think may indeed be cleaner to mark in-place by not returning anything to discriminate it from non--in-place functions.

On the other hand calling a method of a class I think the expectation is anyways that it  will affect the instance - otherwise it should be a standalone function and not a method.
Not having a return self prevents method chaining (i.e. `graph.reverse_edges().set_metadata()`) - as I like the readability of method chaining, I would rather support returning self in such cases.

However this also taste dependent - most important is to stay consistent.